### ISSUE TITLE
Fix qiskit/__init__.py linting

### ISSUE THREAD

**Issue by diego-plan9** [QISKit/qiskit-sdk-py#350]
The style target [is failing](https://travis-ci.org/QISKit/qiskit-sdk-py/jobs/355528602) is failing for `qiskit/__init__.py` due to the recent changes in #349:
```
qiskit/__init__.py:28:1: E402 module level import not at top of file
qiskit/__init__.py:31:1: E402 module level import not at top of file
...
```

It is not being marked as a failure on the travis build due to a bug that will be fixed on the next travis-related PR, but in the meantime, can we make sure both targets pass? [As specified by the contributing guide](https://github.com/QISKit/qiskit-sdk-py/blob/master/CONTRIBUTING.rst#style-guide), we enforce PEP8 conformant code.

Pinging explicitly @nonhermitian and @ajavadia in the hopes of a quick fix, as otherwise the style target will result in an error for the rest of the ongoing PRs and contributors. At a first glance, it seems to be caused by the explicit Python version check that early on the file - which seems conceptually fine, but since it's a bit redundant (we declare the supported Python versions on the setup.py classifiers, the documentation, etc), perhaps moving the check inside `_util` or similar would do the trick?

**Comment by nonhermitian** [comment]
The motivation for this was the removal of the asserts in the tutorials that do the Python version check without losing the safety check.  It was also suggested to move the other checks there as well.  The PEP8 you cite is well known your being a bit too restrictive, and many packages do version and dependency checks before module import.  However, I am not opposed to revertting the whole Pull if people are bothered.

**Comment by diego-plan9** [comment]
Yes, having some safety checks doesn't hurt at all - it's not about reverting the contribution, the issue is that in the current state, the style is not consistent with the project guidelines (and in practice makes the `make style` result in failures in the rest of the open contributions).

What about moving the version check and probably also the call to `_check_ibmqe_version()` from `qiskit/__init__.py` to `qiskit/_util.py` directly, keeping the `_util` import as the first one and "conceptually" expanding it into the version/dependency checker piece within the SDK? This way we can keep being PEP8 conformant, and also have the checks in place.

**Comment by ajavadia** [comment]
@diego-plan9 I thought that including `pylint: disable=wrong-import-position` would silence this warning. And I thought this is fine to add since the `qiskit/__init__.py` file already had `pylint: disable=wrong-import-order` in it.

Did that not make the style errors go away? If not, we can do as you suggest.

**Comment by diego-plan9** [comment]
@ajavadia That line does disable specific linter warnings for that file (which is fine to bypass with both the old `wrong-import-order` and `wrong-import-position` directives -  we actually _need_ to be able to have the imports in non-standard order due to the way we implement the gates at the moment and its dynamic nature), yes! It makes `make lint` happy, but unfortunately the changes on the mentioned PR make the code non PEP8 conformant, which is enforced by the `make style` target.

Can you or @nonhermitian take a look at it? As mentioned, it is causing other open contributions to exhibit errors, and it's also blocking the upcoming PR for tweaks to travis which need to be in place before the release (that, among other things, will make sure that `make style` is handled properly in the CI).

**Comment by ajavadia** [comment]
@diego-plan9 done.

**Comment by diego-plan9** [comment]
Fixed by #355
